core-web/apps/dotcms-ui/src/app/shared/validators/dotValidators.ts

```diff
@@ -30,4 +33,17 @@ export class DotValidators {
             ? null
             : DOT_ERROR_MESSAGES.validQueryParamNameErrorMsg;
     }
+
+    /**
+     * Validate there is not only white spaces in a field when is not empty.
+     *
+     * @param {AbstractControl} control
+     */
+    static noWhitespaceOnly(control: AbstractControl): { [key: string]: string } | null {
+        return control.value
+            ? control.value.trim().length
+                ? null
+                : DOT_ERROR_MESSAGES.whiteSpaceOnlyMgs
+            : null;
+    }
```

Comment:
what a about the name `strictRequired`?

Reply:
I like more `noWhitespace` when you use it in the form is `DotValidators.noWhitespace`